frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -89,3 +90,37 @@ export const alphanumericCompare = (a: string, b: string): number => {
     sensitivity: 'base',
   });
 };
+
+export const translationForResourceKind = {
+  // t('console-shared~Helm Release')
+  HelmRelease: `console-shared~Helm Release`,
+};
+
+export const labelForNodeKind = (kindString: string) => {
+  const model: K8sKind | undefined = modelFor(kindString);
+  if (model) {
+    return model.label;
+  }
+  return startCase(kindString);
+};
+
+export const labelKeyForNodeKind = (kindString: string) => {
+  const model: K8sKind | undefined = modelFor(kindString);
+  if (model) {
+    if (model.labelKey) {
+      return model.labelKey;
+    }
+    return model.label;
+  }
+  if (translationForResourceKind[kindString]) {
+    return translationForResourceKind[kindString];
+  }
+  return startCase(kindString);
+};
+
+export type PageTitleContextValues = {
+  telemetryPrefix?: string;
+  titlePrefix?: string;
+};
+
+export const PageTitleContext = createContext<PageTitleContextValues>({});
```

Comment:
Can you move this info `packages/console-shared/src/components/pagetitle/PageTitleContext.tsx` or `packages/console-shared/src/context/pagetitle/PageTitleContext.tsx`?

I would use context, but other contexts are part of the components folder, see: `packages/console-shared/src/components/toast/ToastContext.tsx`, so that's also fine, but I prefer to not anything new to `src/utils/utils.ts`. You can also move the utils to `packages/console-shared/src/components/pagetitle/utils.tsx` or at least a `page-title-utils.ts` here in the utils folder.

Wdyt?